spec/features/map/access_map_page_spec.rb

```diff
@@ -1,17 +1,17 @@
 require "rails_helper"
 
 RSpec.describe "Map Page", type: :feature do
-  it "is accessible" do
+  it "should exist and render without error" do
```

Comment:
The linter apparently doesn't like us having shouold in the names, the old names seem better

Reply:
done